app/models/finder_schema.rb

```diff
@@ -1,33 +1,69 @@
 class FinderSchema
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  include ActiveModel::Serializers::JSON
+
   # Pluralized names of all document types
   def self.schema_names
     Dir.glob(Rails.root.join("lib/documents/schemas/*.json")).map do |f|
       File.basename(f).gsub(".json", "")
     end
   end
 
-  def self.load_schema_for(type)
-    JSON.parse(File.read(Rails.root.join("lib/documents/schemas/#{type}.json")))
+  def self.load_from_schema(type)
+    new.from_json(File.read(Rails.root.join("lib/documents/schemas/#{type}.json")))
+  end
+
+  attr_writer :editing_organisations, :facets, :organisations, :show_summaries, :taxons
+  attr_accessor :base_path,
```

Comment:
Using `ActiveModel::Model` requires us to specify any attributes we want to use, which is slightly annoying. I considered trying to override some methods to avoid this requirement, but in the end I thought it would be simpler just to list the attributes. There aren't so very many.